ooiui/templates/science/plot.html

```diff
@@ -155,19 +137,22 @@
                                                                 <p class="initial-text"> Please Select an instrument</p>
                                                             </div> <!-- panel-table -->
                                                         </div> <!-- panel-body -->
-                                                        <div class="panel-footer clearfix">
-                                                            <div class="pull-center">
-                                                                <a href="#" class="btn btn-primary update-plot">Plot</a>
-                                                                <small class="pull-right"> * Data is decimated. Maximum of 1000 points shown.</small>
-                                                            </div>
-                                                        </div>
                                                     </div> <!-- panel -->
                                                 </div>
                                             </div>
                                         </div>
                                     </div>
                                 </div>
                             </div>
+                            <div class="panel-footer clearfix">
+                                <div class="pull-left">
+                                    <input id="showAdditionalParameters" type="checkbox" class="" name="Show Additional Parameters" value="showAdditionalParameters"> Show Additional Parameters<br>
+                                </div>
+                                <div class="pull-center">
+                                    <a href="#" class="btn btn-primary update-plot">Plot</a>
+                                    <small class="pull-right">  * Data is decimated. Maximum of 1000 points shown.</small>
```

Comment:
can someone just give me the language to use "* Data are decimated. Maximum of 1000 points shown" is what i have right now